vms/event/services.py

```diff
@@ -1,18 +1,30 @@
 from django.core.exceptions import ObjectDoesNotExist
 from event.models import Event
 
+#check if the event exists and is not empty
+def event_not_empty(event_id):
+    result = True
+    event = get_event_by_id(event_id)
+    if not event:
+        result = False
+    return result
+
 #need to check that this event is not accociated with any jobs, 
```

Comment:
@willingc thank you, I will do it. I was confused about this:
"As with unit tests, for a given Django application, the test runner looks for doctests in two places: The **models.py** file and **tests.py**" (https://docs.djangoproject.com/en/1.5/topics/testing/doctests/)
And there is a file **services.py**